CMakeLists.txt

```diff
@@ -2,9 +2,9 @@ cmake_minimum_required(VERSION 2.8)
 project(KinesisVideoWebRTCClient LANGUAGES C)
 
 # User Flags
-option(BUILD_STATIC "Build with static linkage." OFF)
```

Comment:
Readme should be updated: https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#cmake-arguments